frontend/public/components/storage-class-form.tsx

```diff
@@ -501,25 +551,48 @@ export class StorageClassForm_ extends React.Component<
         },
       },
     },
-  });
+  };
 
   reclaimPolicies = {
     Retain: 'Retain',
     Delete: 'Delete',
   };
 
+  csiProvisionerMap = (csiData) => {
```

Comment:
Can you explain what this code is doing? It's a little hard to follow. This deserves a comment at least.

Reply:
Added comments and used `every` instead of `some`. I want to break if the provisioner is found and forEach and map don't provide that. So that's why using every or some.

Reply:
`_.each` allows you to break by returning false. That would be more clear.

https://lodash.com/docs/4.17.15#forEach